mmdet/core/utils/lr_updater.py

```diff
@@ -0,0 +1,41 @@
+from mmcv.runner import HOOKS
+from mmcv.runner.hooks.lr_updater import CosineAnealingLrUpdaterHook, annealing_cos
+
+@HOOKS.register_module()
+class CosineAnealingLrLastEpochUpdaterHook(CosineAnealingLrUpdaterHook):
```

Comment:
A few thoughts about the name:

- I'd prefer to correctly spell "annealing", but I see the reason of this name choice. Let's put a TODO here, to not forget to fix this at the MMCV version update.
- "LastEpoch" makes an invalid reference to the "last_epoch" parameter of PyTorch LR schedulers and does not give a sufficient explanation of what does it have influence on. "UntilEpoch" sounds better to me in this regard.

Reply:
- You are right about unsuitable naming. I agree with your second point.
- About the "annealing" spelling: I do not really understand why it is "anealing" in MMCV, but I'm free to use the correct version here. It will be inconsistent but right.

Reply:
This is simply a typo, and it's been fixed in MMCV a while ago. But we're still limited in the use of the latest versions of it. So, my suggestion was (and still is) to keep the naming aligned with the one used in supported version of MMCV, but to add a TODO/FIXME comment to change it once the upgrade of MMCV become possible.